scylladb/sphinx-scylladb-theme
# [Search page] Style new projects component

Closes #299

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/getting-started/#q=aaa to preview the changes:

![image](https://user-images.githubusercontent.com/9107969/149141605-cb15b7cf-f8bb-4426-9060-c55691a277d3.png)
